tests/unit/test__versions_helpers.py

```diff
@@ -173,3 +178,64 @@ def test_bqstorage_is_read_session_optional_false():
     bqstorage_versions = _versions_helpers.BQStorageVersions()
     with mock.patch("google.cloud.bigquery_storage.__version__", new="2.5.0"):
         assert not bqstorage_versions.is_read_session_optional
+
+
+@pytest.mark.skipif(pandas is None, reason="pandas is not installed")
+def test_try_import_raises_no_error_w_recent_pandas():
+    versions = _versions_helpers.PandasVersions()
+    with mock.patch("pandas.__version__", new="1.5.0"):
+        try:
+            pandas = versions.try_import(raise_if_error=True)
+            assert pandas is not None
+        except exceptions.LegacyPandasError:  # pragma: NO COVER
+            raise ("Legacy error raised with a non-legacy dependency version.")
+
+
+@pytest.mark.skipif(pandas is None, reason="pandas is not installed")
+def test_try_import_returns_none_w_legacy_pandas():
+    versions = _versions_helpers.PandasVersions()
+    with mock.patch("pandas.__version__", new="1.0.0"):
+        pandas = versions.try_import()
+        assert pandas is None
+
+
+@pytest.mark.skipif(pandas is None, reason="pandas is not installed")
+def test_try_import_raises_error_w_legacy_pandas():
+    versions = _versions_helpers.PandasVersions()
+    with mock.patch("pandas.__version__", new="1.0.0"):
+        with pytest.raises(exceptions.LegacyPandasError):
+            versions.try_import(raise_if_error=True)
+
+
+@pytest.mark.skipif(pandas is None, reason="pandas is not installed")
+def test_try_import_returns_none_w_newer_pandas():
+    versions = _versions_helpers.PandasVersions()
+    with mock.patch("pandas.__version__", new="2.0.0"):
+        pandas = versions.try_import()
+        assert pandas is None
```

Comment:
Let's actually test that it's not `None` :-)

Reply:
I am not sure what you meant, I have a test above this one that tests if it raises no error if its a supported version f pandas already, if that is what you meant

Reply:
Yes. I want a test where we check for some 2.x version.

Reply:
Implemented fix through pytest.parameterize in newest commit